eventbridge/src/main/scala/akka/stream/alpakka/eventbridge/javadsl/EventBridgePublisher.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.eventbridge.javadsl
+
+import java.util.concurrent.CompletionStage
+
+import akka.stream.alpakka.eventbridge.EventBridgePublishSettings
+import akka.stream.javadsl.{Flow, Keep, Sink}
+import akka.{Done, NotUsed}
+import software.amazon.awssdk.services.eventbridge.EventBridgeAsyncClient
+import software.amazon.awssdk.services.eventbridge.model.{PutEventsRequest, PutEventsResponse}
+
+/**
+ * Scala API
+ * Amazon EventBridge publisher factory.
+ */
+object EventBridgePublisher {
+
+  /**
+   * creates a [[akka.stream.scaladsl.Flow Flow]] to publish event to a EventBridge EventBus using an [[software.amazon.awssdk.services.eventbridge.EventBridgeAsyncClient EventBridgeAsyncClient]]
+   */
+  def createFlow(eventBusName: String,
```

Comment:
Optionally an overloaded method could be introduced that doesn't require to specify the eventbus name, this means that events will be publish to `default` event bus.